### DISCUSSION TITLE
Favorite enemy?

### DISCUSSION THREAD

**Discussion by ExplodingHairyPotatoCat** [geckoking7/kgt_idle#15] [General]
I  want to know what your guys's favorite enemies are. Mine personally is the blurred out among us crewmates because I think it's funny :)

**Comment by geckoking7** [comment]
So having a blurred out among us crewmate wasn't to out of line?
Well my favourite would probably the 'Gasby gaming'
Also, do you have any big brain ideas for the 11th zone?

**Reply by ExplodingHairyPotatoCat** [reply]
Funnily enough, I think that you could have a giant floating brain be like a continuation of the UFO type enemies. What are you having as the theme for zone 11?

**Reply by geckoking7** [reply]
Z11 is the one that connects from the coders lab and the mines, so I was thing some maybe underground supercomputer like thing. So I guess a super brain could fit.

**Reply by ExplodingHairyPotatoCat** [reply]
For a much less serious suggestion, you could have Gordon Freeman since black mesa is like an underground super computer facility thing (or you could make it like portal/black mesa themed)

**Reply by geckoking7** [reply]
I could do something like that, it just I've never played half life

**Reply by ExplodingHairyPotatoCat** [reply]
I've played black mesa (half-life 1 remake) so I could help if you do decide to do that

**Reply by geckoking7** [reply]
I'll probably have something similar to it where Gordon is a reference. Thanks anyway
